examples/gcoap/client.c

```diff
@@ -338,9 +370,20 @@ int gcoap_cli_cmd(int argc, char **argv)
         return 0;
     }
     else {
-        printf("usage: %s <get|post|put> [-c] <addr>[%%iface] <port> <path> [data]\n",
+#ifdef SOCK_HAS_IPV4
+        printf("usage: %s <get|post|put> [-c] <IPv4 addr:port> <path> [data]\n",
+               argv[0]);
+#endif
+#ifdef SOCK_HAS_IPV6
+        printf("usage: %s <get|post|put> [-c] <[[IPv6 addr[%%iface]]]:port> <path> [data]\n",
                argv[0]);
-        printf("       %s ping <addr>[%%iface] <port>\n", argv[0]);
+#endif
+#ifdef SOCK_HAS_IPV4
+        printf("       %s ping <IPv4 addr:port>\n", argv[0]);
+#endif
+#ifdef SOCK_HAS_IPV6
+        printf("       %s ping <[[IPv6 addr[%%iface]]]:port>\n", argv[0]);
+#endif
```

Comment:
```suggestion
        printf("       %s ping <host:port>\n", argv[0]);
```

let's keep it simple